Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -14,7 +14,7 @@ final class WebSocket {
     static let pingTimeInterval: TimeInterval = 25
     
     /// A WebSocket connection callback.
-    private let onEvent: (Event) -> Void
+    private let onEvent: (Event) -> Bool
```

Comment:
Why does this need a return value? Can you add a comment explaining that?

Reply:
I added a note in the description: _I moved a logic related to the current user from WebSocket to the Client to make tests work._

Reply:
I'm talking about the return value of the `onEvent` callback. It now returns `Bool`. What doesn't the return value indicate?

Reply:
It needs for the Client to decide should an event be sent to subscribers:
https://github.com/GetStream/stream-chat-swift/pull/282/files#diff-f4d681bd989cb18a98bd77ba7245295bR213-R218

Reply:
>  Can you add a comment explaining that?